testing/conformance/tests/runner.rs

```diff
@@ -190,11 +190,13 @@ fn compare_state_roots(bs: &MemoryBlockstore, root: &Cid, vector: &MessageVector
 async fn conformance_test_runner() -> anyhow::Result<()> {
     pretty_env_logger::init();
 
+    let engine = Engine::default();
```

Comment:
For M1, do we intend to have Lotus create a single engine for the full process lifetime?

Reply:
Yes, the FFI branch currently does that (stores it in a lazy static).